### DISCUSSION TITLE
How to redirect event?

### DISCUSSION THREAD

**Discussion by HppZ** [microsoft/WindowsAppSDK#1173] [Q&A] [answered]
How to redirect event (such as ScrollEvent) to ListView even my pointer is not on ListView?

**Comment by btueffers** [comment, marked as answer]
@HppZ this sounds like a good question for the WinUI team. They would be able to provide more insight if you opened an Issue in their repo 😊 https://github.com/microsoft/microsoft-ui-xaml
